platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/exceptions/InvalidAccessTokenException.java

```diff
@@ -13,7 +13,7 @@
 public class InvalidAccessTokenException extends RuntimeException {
 
     public InvalidAccessTokenException() {
-        super("Using MapView requires setting a valid access token. See the INSTALL.md");
+        super("\nUsing MapView requires setting a valid access token. Please see https://www.mapbox.com/help/create-api-access-token/ to learn how to create one.\n     Lastly, follow the steps in this guide https://www.mapbox.com/help/first-steps-android-sdk/#access-tokens in order to include the access token in your app.");
```

Comment:
Should be updated to follow the 3 steps:

1. State the problem
1. State the technical solution
1. For more help click here...